.github/workflows/commercial-e2e.yml

```diff
@@ -4,6 +4,7 @@ on:
         paths:
             - static/src/javascripts/projects/common/modules/commercial/**
             - static/src/javascripts/projects/commercial/**
+            - static/src/javascripts/projects/common/modules/spacefinder-*
```

Comment:
Is the idea to add some tests for debug mode?

Reply:
I thought it best to run tests if spacefindery stuff changes